### ISSUE TITLE
Adding a red bar section for again

### ISSUE THREAD

**Issue by DorianK29** [fabd/kanji-koohii#234]
![image](https://user-images.githubusercontent.com/80626896/147816170-739b3599-5620-461e-ab90-6ee59188b178.png)
Adding a red bar next to the green bar for every kanji selected as again. In this picture I have done 18 kanji but the bar shows as 15/22, it would be a good feature if it showed 15/22 as green and 3/22 as red for every kanji selected again, and when again is changed to yes or easy it should change to green.

**Comment by fabd** [comment]
I agree, the review status area on the Flashcard Review page is not good since I added Again. It needed some kind of count but it was put together without a design step.

I'll pin this so I don't forget it as I did intend to improve it as well as the little chart on the review summary. I think both need to do a better job of separating No / Hard / Yes+Easy instead of just Fail / Pass. Then, the progress bar indeed could use a better design.
